model/src/lib.rs

```diff
@@ -166,6 +165,10 @@ impl Leverage {
     pub fn get(&self) -> u8 {
         self.0
     }
+
+    pub const ONE: Self = Self { 0: 1 };
```

Comment:
Why this weird syntax of using a named field syntax when you have a tuple struct?

```suggestion
    pub const ONE: Self = Self(1);
```

That doesn't work?